tests/jetty-jmh/pom.xml

```diff
@@ -72,6 +72,11 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>org.openjdk.jol</groupId>
+      <artifactId>jol-core</artifactId>
+      <version>0.14</version>
```

Comment:
The version should be moved up in the root pom's `dependencyManagement`. But it also looks like you're not sizing objects in the benchmark you added - is that an overlook or something you don't care about?

Reply:
it wasn't even needed